### PR TITLE
Bug 1454929 openshift_metrics_hawkular_hostname is no longer a required variable

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1454929

### CODE REVIEW

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :icecream:

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -439,6 +439,12 @@ pods are placed onto nodes with specific labels. For example,
 remove the HOSA from an installation. HOSA can be used to collect custom
 metrics from your pods. This component is currently in
 Technology Preview and is not installed by default.
+
+|`openshift_metrics_hawkular_hostname`
+|Set `openshift_metrics_hawkular_hostname` when executing the
```
Comment (ahardin-rh) [inline install_config/cluster_metrics.adoc]: Maybe just drop the name in the description and just say "Set when executing..."

Comment (bmcelvee) [reply install_config/cluster_metrics.adoc]: Done, thanks! :dog:

**Pull request merged**